packages/ra-no-code/src/ApplicationsDashboard/ApplicationsDashboard.tsx

```diff
@@ -0,0 +1,128 @@
+                alt="react-admin logo"
+            />
+            <Typography component="h2" variant="h2" className={classes.title}>
+                Welcome to React-admin
+            </Typography>
+            <NewApplicationForm
+                applications={applications}
+                onApplicationCreated={handleApplicationCreated}
+            />
+            {applications.length > 0 && (
+                <Card className={classes.applications}>
+                    <List>
+                        {applications.map(application => (
+                            <ListItem
+                                key={application.name}
+                                button
+                                onClick={() =>
+                                    onApplicationSelected(application)
+                                }
+                            >
+                                <ListItemAvatar>
+                                    <Avatar>
+                                        <FolderIcon />
+                                    </Avatar>
+                                </ListItemAvatar>
+                                <ListItemText
+                                    primary={application.name}
+                                    secondary={new Date(
+                                        application.created_at
+                                    ).toLocaleDateString()}
+                                />
+                            </ListItem>
+                        ))}
+                    </List>
+                </Card>
+            )}
+        </Container>
+    );
+};
+
+const loadApplicationsFromStorage = () => {
```

Comment:
This and the following method are the "model" of the app storage, and as such I think they should live in a dedicated file